product_docs/docs/epas/11/epas_compat_sql/21_create_public_database_link.mdx

```diff
@@ -30,17 +30,19 @@ When the `CREATE DATABASE LINK` command is given, the database link name and the
 
 A SQL command containing a reference to a database link must be issued while connected to the local database. When the SQL command is executed, the appropriate authentication and connection is made to the remote database to access the table or view to which the `@dblink` reference is appended.
 
-!!! Note
-    A database link cannot be used to access a remote database within a standby database server. Standby database servers are used for high availability, load balancing, and replication.
+!!!note "Oracle compatibility"
+-  For EDB Postgres Advanced Server 15, the CREATE DATABASE LINK command has been tested and certified with all the minor versions for use with Oracle versions 10g Release 2, 11g Release 2, 12c Release 1, 18c Release 1, 19c, and 23.
```

Comment:
This note reads "15" in versions 11 through 16 - it should probably correspond to the actual version being modified. In addition, per Tushar's comment on DB-2625, this has only been tested on v12 and above (which is in keeping with our official list of supported versions of EPAS) - so we shouldn't be adding this note to 11 at all.